widget/embedded/src/components/WalletTransactionTypesModal/WalletTransactionTypesModal.tsx

```diff
@@ -0,0 +1,98 @@
+import type { TransactionType } from 'rango-sdk';
```

Comment:
If only one namespace can be selected (single mode), we should show radio button instead of checkbox.

Reply:
Fixed